.github/workflows/ci.yml

```diff
@@ -279,25 +279,59 @@ jobs:
       BUILD_DIR: ${{ needs.build_all.outputs.build-dir }}
       BUILD_DIR_ARCHIVE: ${{ needs.build_all.outputs.build-dir-archive }}
       BUILD_DIR_GCS_ARTIFACT: ${{ needs.build_all.outputs.build-dir-gcs-artifact }}
-    strategy:
-      matrix:
-        target:
-          - runner-type: gpu
-            iree-ctest-label-regex: ^requires-gpu|^driver=vulkan$|^driver=cuda$
-            iree-nvidia-sm80-tests-disable: 1
-            iree-multi-device-tests-disable: 1
-          - runner-type: a100
-            iree-ctest-label-regex: ^requires-gpu-sm80$
-            iree-nvidia-sm80-tests-disable: 0
-            iree-multi-device-tests-disable: 1
-      # Run other jobs even if one fails.
-      fail-fast: false
-    name: test_${{ matrix.target.runner-type }}
     runs-on:
       - self-hosted # must come first
       - runner-group=${{ needs.setup.outputs.runner-group }}
       - environment=${{ needs.setup.outputs.runner-env }}
-      - ${{ matrix.target.runner-type }}
+      - gpu
+      - os-family=Linux
+    steps:
+      - name: "Checking out repository"
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+        with:
+          submodules: true
+      - name: Querying GPU information
+        run: |
+          ./build_tools/scripts/check_cuda.sh
+          ./build_tools/scripts/check_vulkan.sh
+      - name: "Downloading build dir archive"
+        run: gcloud storage cp "${BUILD_DIR_GCS_ARTIFACT}" "${BUILD_DIR_ARCHIVE}"
+      - name: "Extracting build dir archive"
+        run: tar -xf "${BUILD_DIR_ARCHIVE}"
+      - name: "Testing all"
+        env:
+          IREE_NVIDIA_SM80_TESTS_DISABLE: "1"
+          IREE_MULTI_DEVICE_TESTS_DISABLE: "1"
+          IREE_CTEST_LABEL_REGEX: "^requires-gpu|^driver=vulkan$|^driver=cuda$"
```

Comment:
Are the quotes here necessary / correct? The matrix didn't have quotes before, and yml / GitHub Actions can be pretty particular about syntax.

Reply:
IIUC it's just the different style of yaml string (https://yaml.org/spec/1.2.2/#double-quoted-style) but seems like we don't use double quotes in most of the places. Removed them.